license.txt

```diff
@@ -0,0 +1,26 @@
+Copyright (c) <year> <copyright holders>

```

Comment:
What's wrong with the existing license?
https://github.com/tan-tan-kanarek/github-php-client/blob/master/license.txt